app/helpers/sessions_helper.rb

```diff
@@ -40,4 +44,44 @@ def remember user
     cookies.permanent.signed[:user_id] = user.id
     cookies.permanent[:remember_token] = user.remember_token
   end
+
+  def remember_cookies user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
+  def remember_session user
+    user.remember
+    session[:remember_token] = user.remember_token
+  end
+
+  def store_location
+    session[:forwarding_url] = request.original_url if request.get?
+  end
+
+  private
+
+  def find_user_by_session
+    return unless (user_id = session[:user_id])
+
+    user = User.find_by(id: user_id)
+    return unless user
+
+    return unless user.authenticated?(session[:remember_token])
+
+    user
```

Comment:
```suggestion
    return unless user_id = session[:user_id]
    return unless user = User.find_by(id: user_id)
    return unless user.authenticated?(session[:remember_token])

    user
```
các return if liên tục có thể không cần line trống

Reply:
done